_analyzers/normalizers.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Normalizers
+nav_order: 100
+---
+
+# Normalizers
+
+A _normalizer_ has similar function to an analyzer but it outputs only a single token. It does not contain a tokenizer and it can only include specific types of character filters and token filters. These filters can perform only character-level operations, such as character or pattern replacement, but cannot operate on the whole-token level. This means that replacing a token with a synonym or stemming is not supported.
+
+A normalizer is useful in keyword search (that is, in term-based queries) because it enables running token filters and character filters on given input. For instance, it makes it possible to match an incoming query `Naïve` with the index term `naive`.
+
+Consider the following example.
+
+Create a new index with custom normalizer:
+```json
+PUT /sample-index
+{
+  "settings": {
+    "analysis": {
+      "normalizer": {
+        "normalized_keyword": {
+          "type": "custom",
+          "char_filter": [],
+          "filter": [ "asciifolding", "lowercase" ]
+        }
+      }
+    }
+  },
+  "mappings": {
+    "properties": {
+      "approach": {
+        "type": "keyword",
+        "normalizer": "normalized_keyword"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Index document:
+```json
+POST /sample-index/_doc/
+{
+  "approach": "naive"
+}
+```
+{% include copy-curl.html %}
+
+The following query matches the document. This is expected:
+```json
+GET /sample-index/_search
+{
+  "query": {
+    "term": {
+      "approach": "naive"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+But this query matches the document as well:
+```json
+GET /sample-index/_search
+{
+  "query": {
+    "term": {
+      "approach": "Naïve"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+To understand why, try to see the effect of the normalizer:
+```json
+GET /sample-index/_analyze
+{
+  "normalizer" : "normalized_keyword",
+  "text" : "Naïve"
+}
+```
+
+Internally, a normalizer accepts only filters that are instances of either `NormalizingTokenFilterFactory` or `NormalizingCharFilterFactory`. The following is a list of compatible filters found in modules and plugins that are part of the core OpenSearch repository:
+
+### Module `common-analysis`
```

Comment:
@kolchfa-aws As with the "Step" headings in an earlier PR, the word order of this and the following three headings is confusing. Please revise.

Reply:
@kolchfa-aws I think we can resolve this one too now.